### DISCUSSION TITLE
img old header information after JPG conversion from CYMK to RGB need to update / refresh...

### DISCUSSION THREAD

**Discussion by GioMBG** [ImageMagick/ImageMagick#1913] [General]
Hi ALL,
after successfully convert JPG images from CYMK to RGB I ( suppose ) to continue reading old img header infos when was CYMK...
any suggestion in how to rebuild or convert the imgs header info to his actual status ?
thanks
Gio

**Comment by urban-warrior** [comment]
Can you restate your question?  Your post is not clear what you are looking for.  Post the command you are using, what is produced by your command, and how the results differs from what you expect.
